shapes/Square.cpp

```diff
@@ -7,6 +7,9 @@ Square::Square(Color c)
 Square::Square(double x, Color c)
     : Rectangle(x, x, c) {}
 
+Square::Square(double x)
+    : Rectangle(x, x, Color::Red) {}
```

Comment:
this one is just a base class constructor call. Delegation takes place when you call other constructor of your own class.